### PR TITLE
upgrade of all npm dependencies

### PR DESCRIPTION
**Pull request by capricube**
only had to change N3Util.isUrl() call to N3Util.isIRI()

### CODE REVIEW

**Comment by capricube** [comment]
i'm having a strange "works on my machine" moment ... need to investigate a bit more

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- 0.10
+- 5
```
Comment (mcollina) [inline .travis.yml]: This should be 0.10, 0.12, 4, 5

**Comment by mcollina** [comment]
Let me know when it's done :).

**Comment by capricube** [comment]
finally i found the problem. it was a hard nut to crack!

FYI: in order to make the tests work with the updated dependencies i had to downgrade level-test from 2.0.1 to 1.7.0. The tests didn't work on travis because they where executed using the in-memory leveldown (memdown) implementation. When i executed it locally i always had a leveldown dependency and everything worked. Only after i changed the `test/helper.js` file i was able to reproduce the travis errors locally. However i cannot say what the exact reason is why it doesn't work with memdown. I also added leveldown as devDependency so memdown is not being used.

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- 0.10
+  - "5"
```
Comment (mcollina) [inline .travis.yml]: this should be 0.10, 0.12, 4, 5

File: package.json [hunk 3]
```diff
@@ -32,24 +32,25 @@
...
+    "chai": "3.5.0",
+    "coveralls": "2.11.8",
+    "istanbul": "0.4.2",
+    "level-test": "1.7.0",
+    "leveldown": "^1.4.4",
+    "levelgraph": "1.1.1",
+    "lodash": "2.4.1",
+    "mocha": "2.4.5",
+    "uglify-js": "2.6.2",
+    "zuul": "3.10.1"
```
Comment (mcollina) [inline package.json]: can you please restore all the dependencies to be with `~`?

File: test/iri_spec.js [hunk 4]
```diff
@@ -37,7 +37,7 @@ describe('IRI', function() {
         done();
       });
     });
-  });
+  });
```
Comment (mcollina) [inline test/iri_spec.js]: unneeded space

**Comment by mcollina** [comment]
Good work! There are just a couple of nits to fix.

**Comment by capricube** [comment]
do we really need support for node 0.10 and 0.12? Those two are failing...

**Comment by mcollina** [comment]
0.10 and 0.12 are currently supported by this library and by levelgraph (https://github.com/mcollina/levelgraph/blob/master/.travis.yml)

So, we should keep supporting them.

**Comment by capricube** [comment]
if you merge my pull request for levelgraph i can upgrade the levelgraph depencency and the build will be fixed.

**Comment by mcollina** [comment]
I've merged and released the levelgraph dep.

**Comment by mcollina** [comment]
Done but tests are now failing everywhere apart from v5.

**Comment by capricube** [comment]
will test again today in the evening

**Pull request merged**